meson.build

```diff
@@ -223,6 +223,7 @@ endif
 endif
 libjcat = dependency('jcat', version : '>= 0.1.0', fallback : ['libjcat', 'libjcat_dep'])
 libjsonglib = dependency('json-glib-1.0', version : '>= 1.1.1')
+protobufc = dependency('libprotobuf-c')
```

Comment:
this makes it a hard dependency, I think you want something in meson_options so we can have `-Dprotobuf=false` for the CI targets like Windows where this dep doesn't make sense.

Reply:
Working on it